code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -249,23 +249,26 @@
 				user.visible_message("<span class='warning'>[user]'s palms flare a bright red against [target]'s temples!</span>")
 				to_chat(target, "<span class='danger'>A terrible red light floods your mind. You collapse as conscious thought is wiped away.</span>")
 				target.Weaken(12)
-				sleep(20)
+				sleep(2 SECONDS)
 				if(ismindshielded(target))
-					to_chat(user, "<span class='notice'>They have a mindshield implant. You begin to deactivate it - this will take some time.</span>")
+					to_chat(user, "<span class='notice'>They have a mindshield implant. You try to overwhelm it!</span>")
 					user.visible_message("<span class='warning'>[user] pauses, then dips [user.p_their()] head in concentration!</span>")
 					to_chat(target, "<span class='boldannounce'>Your mindshield implant becomes hot as it comes under attack!</span>")
-					sleep(100) //10 seconds - not spawn() so the enthralling takes longer
-					to_chat(user, "<span class='notice'>The nanobots composing the mindshield implant have been rendered inert. Now to continue.</span>")
-					user.visible_message("<span class='warning'>[user] relaxes again.</span>")
-					for(var/obj/item/implant/mindshield/L in target)
+					sleep(10 SECONDS) //10 seconds - not spawn() so the enthralling takes longer
+					to_chat(user, "<span class='notice'>The nanobots composing the mindshield implant put up a valiant fight but they are about to lose...</span>")
+					to_chat(user, "<span class='boldannounce'>Rather than be disabled, the mindshield superheats, frying your victim's brain!</span>")
+					to_chat(target, "<span class='boldannounce'>You feel a burning, horrible pain in your head!</span>")
+					for(var/obj/item/implant/mindshield/L in target)	//it self-destructs to kill you, after all
 						if(L && L.implanted)
 							qdel(L)
-					to_chat(target, "<span class='boldannounce'>Your mental protection implant unexpectedly falters, dims, dies.</span>")
+					target.adjustBrainLoss(120, TRUE, FALSE)
```

Comment:
Im really not sure that giving the target **120** brain damage is a good solution here

Reply:
120 is the brain damage needed for instant death. If I did less damage than that they either wouldn't die or (if I manually call death() after applying it) they could simply be defibbed back alive.

I considered doing a lot of burn damage to the head instead but this runs into the same issue. I'd have to either make it a ridiculous number (like 500 burn or whatever) to more or less guarantee death or I'd have to manually call death(), at which point you just slap some burn patches on and defib.

Someone also brought up the option of exploding their head, but that would make them harder to clone as well and I didn't want found, dead sec officers to be out for longer than the normal clone timer.

If you have a good idea for a solution, I'm all ears.

Reply:
Currently, it takes 19 seconds without the target moving to kill them, which is shorter than the regular enthrall time, and a lot more potent (Slings can now wipeout security officers in a smaller time than enthralling, then stash the body in the depths of maintenance, greatly hindering security)

Im not sure death is even a good solution here, maybe just denying the enthrall in the same way that cult and vamp thralls do.

Reply:
Yeah, probably best to just make mindshields block thralling, instead.